src/shared/utils/index.js

```diff
@@ -89,7 +107,9 @@ const setChartMargin = (
   maxYAxisTickLabelWidth,
   lastXAxisTickLabelWidth,
   maxRowLegendItems,
-  trimLegend
+  trimLegend,
+  TEXT_HEIGHT,
+  LEGEND_HEIGHT
```

Comment:
_pref:_ these are no longer constants, so change text case

Reply:
Hmmm...they are still constants. TEXT_HEIGHT is only set once and doesn't change value afterwards. LEGEND_TEXT, for now, is equal to it and also doesn't change. So, in my opinion, they are both constants.

Reply:
if they're passed in as parameters, they're no longer _global_ constants. that's what the ALL_CAPITAL case is used for, conventionally.

Reply:
ok, wasn't clear about that...